aws/network-primitives/instances.tf

```diff
@@ -2,8 +2,8 @@ resource "aws_instance" "bootstrap_node" {
   count             = length(var.aws_region) * var.bootstrap-node-config.instance-count
   ami               = data.aws_ami.ubuntu_amd64.image_id
   instance_type     = var.instance_type
-  subnet_id         = element(aws_subnet.public_subnets.*.id, count.index)
-  availability_zone = element(var.azs, count.index)
+  subnet_id         = element(aws_subnet.public_subnets.*.id, 0)
+  availability_zone = element(var.azs, 0)
```

Comment:
does Az has to be the first one always ?
if so, why not just make it a single value instead of list ?

Reply:
Yeah can make it a fixed string since we use one now.